scripts/metadata_comparison/metadata_comparison/extractor.py

```diff
@@ -23,18 +23,11 @@
 from google.cloud import storage
 import google.auth
 from googleapiclient.discovery import build as google_client_build
-import logging as log
+from metadata_comparison.lib.logging import *
```

Comment:
non-`*`

Reply:
Thanks. I want like to minimize impact on the other parts of `metadata_comparison`, so I'll revert the changes which I made for extractor.py